portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,17 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Chooses a random cat photo to display on page
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function changeCatPhoto() {
+  const photos =
+      ["/images/cats-1.jpg", "/images/cats-2.jpg", "/images/cats-3.jpg", "/images/cats-4.JPG", "/images/cats-5.jpg"];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random cat photo.
+  const photo = photos[Math.floor(Math.random() * photos.length)];
+  console.log("changed photo to " + photo);
```

Comment:
nit: is there an explicit reason for this log? Generally, logs are only added to code when you want to debug or track metrics

Reply:
I originally had a bug with changing the cat photos here, but then I figured it out (partially using the log) and fixed it. Since I fixed it is it common practice to get rid of the console.log now?

Reply:
Yep! It's generally a good idea to remove any logs that are not actively contributing to a metric in prod (ex: let's say you're tracking how many times people change cat photos, then it would probably be fine to leave it). If logs were specifically used for debugging, they are usually removed even before submitting a PR (since you'd only want to submit a PR if you have functioning code).